test/interval_tests/complex.jl

```diff
@@ -14,7 +14,7 @@ end
 
 @testset "Complex functions" begin
```

Comment:
Using this PR I get the following:
```julia
julia> 0.2..0.2
Interval(0.19999999999999998, 0.20000000000000004)

julia> 0.2 ± 0
Interval(0.19999999999999998, 0.20000000000000004)

julia> @interval(0.2)
Interval(0.19999999999999998, 0.2)
```

Current master (for all three versions) yields `Interval(0.19999999999999998, 0.2)
`. My point is that `0.2 ± 0.0` should behave as `@interval` and not as `0.2..0.2`.